src/main/resources/org/jenkinsci/plugins/configfiles/maven/GlobalMavenSettingsConfig/edit-config.jelly

```diff
@@ -26,15 +26,7 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-<style type="text/css">
-div.credentialssection {
-  border-bottom: 1px solid black;
-}
-</style>
 			<j:set var="instance" value="${config}"/>
-	   		<p>
-	   		 ${%description}
```

Comment:
FTR there is no localized text for this, so it makes sense to remove without replacement:
<img width="662" alt="Screenshot 2022-01-08 at 23 08 15" src="https://user-images.githubusercontent.com/1831569/148661578-c8ce4ca1-caa5-4f17-b01e-e2d19bad29b6.png">